assets/grafana/node-dashboard.json

```diff
@@ -405,15 +414,15 @@
           },
           "targets": [
             {
-              "expr": "(node_memory_MemFree{instance=~\"$server\"} / node_memory_MemTotal{instance=~\"$server\"}) * 100",
```

Comment:
This works?

I assume `$server` is the variable interpolated by Grafana. All regex matches are anchored in Prometheus though. So `=~` would be equivalent to `=` here unless `$server` contains regex chars itself, which it shouldn't though that's not escaped by anything.

If it's about allowing a port suffix, this should only work when `.+` or similar is added to the variable. Though I don't know when Grafana stops lexing the var here.